dotnet/src/Connectors/Connectors.Memory.Pinecone/Model/PodType.cs

```diff
@@ -96,7 +96,13 @@ public enum PodType
     /// Enum Starter for value: starter
     /// </summary>
     [EnumMember(Value = "starter")]
-    Starter = 13
+    Starter = 13,
```

Comment:
@Imir From PR description:
```
The problem is related to Pinecone connector’s inability to deserialize the “nano” pod type, which is the free tier of Pinecone. Previously the pod type for free tier was “starter”.
```

Does it mean that `starter` pod type doesn't exist anymore and instead of adding new enum value `Nano`, we should rename `Starter` to `Nano`?

Reply:
It’s a valid observation. A few weeks ago, my current `starter` pod began returning `nano` as its pod type. This change is also reflected in Pinecone’s dashboard. After your comment, I created new “free” indexes using both `starter` and `nano` pod types for testing purposes. Both variations function as expected, although the dashboard consistently displays the pod type as `starter`. It looks like the `starter` pod is still valid, and I believe it’s necessary to retain both.

Reply:
That makes sense, thank you for explanation!